app/src/CMakeLists.txt

```diff
@@ -19,10 +19,6 @@ add_executable(${TARGET_NAME}
     Launcher.cpp
 )
 
-add_dependencies(${TARGET_NAME}
-    vehicle-app-sdk
-)
-
 target_link_libraries(${TARGET_NAME}
-    vehicle-app-sdk
+    ${CONAN_LIBS}
```

Comment:
is this just the sdk or are we now linking against all conan-libs which we install?

Reply:
all unused dependencies are stripped when building the container